lib/Review.php

```diff
@@ -22,12 +22,25 @@
  *
  * @package Stripe
  */
-class Review extends \Stripe\ApiResource
+class Review extends ApiResource
```

Comment:
None of the other resources have fully qualified namespaces here, so I removed it.